app/src/main/res/values/strings.xml

```diff
@@ -520,7 +520,7 @@
     <string name="pref_remove_all_rules">Remove all rules</string>
     <string name="pref_remove_all_rules_msg">Permissions will be granted, app ops and components will return to their default values.</string>
     <string name="are_you_sure">Are you sure?</string>
-    <string name="filter_apps_with_activities">Apps with activities</string>
+    <string name="filter_apps_with_activities">With activities</string>
```

Comment:
Filter names do not seem to be consistent. Check if all filters are named in the correct form.

Reply:
Consistent now, but do you want "Apps with __" for all of them instead?

Reply:
No, since it was agreed that the _filter_ will be replaced with _filter apps_.